tests/DqtApi.Tests/DataverseIntegration/SetIttOutcomeForTeacherTests.cs

```diff
@@ -215,7 +215,47 @@ public void SelectIttRecord_given_multiple_matching_itt_records_returns_Multiple
             Assert.Null(failedReason);
         }
 
-        // SelectQtsRecord
+        [Fact]
```

Comment:
Would be good to have some negative tests here to prove that the field is *not* nulled out for a non-withdrawn status